apps/server/src/modules/health/repo/entity/healthcheck.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, PrimaryKeyType, PrimaryKey, Property, Index } from '@mikro-orm/core';
+
+export interface HealthcheckEntityProps {
+	id: string;
+	updatedAt: Date;
+}
+
+@Entity({ tableName: 'healthchecks' })
+export class HealthcheckEntity {
```

Comment:
```suggestion
export interface HealthCheckEntityProps {
	id: string;
	updatedAt: Date;
}

@Entity({ tableName: 'healthchecks' })
export class HealthCheckEntity {
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/82f1143936a6bcf40d27ca83ebbaa47c6582b9d0 .